qemu/tests/qemu_guest_agent.py

```diff
@@ -215,7 +215,7 @@ def gagent_verify(self, params, vm):
         if not self.gagent:
             self.test.error("Could not find guest agent object "
                             "for VM '%s'" % vm.name)
-
+        time.sleep(10)
```

Comment:
why sleep here? better to use `wait_for`.

Reply:
the sleep is useless. delete it.